views/familyCalendar/familyCalendar.js

```diff
@@ -144,19 +152,46 @@ window.CalendarView = class CalendarView extends View {
                                     originalHeaderRight.style.display = '';
                                 });
                             }
+                        },
+                        listDay: {
+                            text: 'day',
+                            click: function () {
+                                $('#view-container').fullCalendar('changeView', 'listDay');
+                                $('.fc-customButton').removeClass('active-view-button');
+                                $('.fc-listDay-button').addClass('active-view-button');
```

Comment:
I suggest adding here:

$("td.fc-list-item-time").css("display", "none");

All of these items are "all day", so displaying the time column is unnecessary.